content/kb/troubleshooting-restore-pvc-stuck-using-velero-csi-plugin-version-lower-than-0.4.md

```diff
@@ -0,0 +1,36 @@
+---
+title: "Troubleshooting: Restoring PersistentVolumeClaim is stuck in the Pending state via Velero CSI Plugin < 0.4"
+author: Ray Chang
+draft: false
+date: 2022-12-15
+categories:
+  - "restore"
+---
+
+## Applicable versions
+
+All Longhorn versions.
+
+## Symptoms
+
+When using Velero with Velero CSI Plugin (< 0.4) to restore PersistentVolumeClaim manipulated by Longhorn, the PVC will be stuck in the `Pending` state.
```

Comment:
```suggestion
PersistentVolumeClaim is stuck in the `Pending` state when restoring Longhorn with Velero with Velero CSI Plugin version prior to v0.4.0.
```

Reply:
Thank you. I have updated the description as per the suggestion.